server/vars.py

```diff
@@ -1,6 +1,7 @@
 import os
 
 ACQUIRE_TIMEOUT = 3  # The timeout used to time out a locking-request
+DELIMITER = ","  # delimiter to parse csv strings
```

Comment:
Maybe rename it to `RESULT_DELIMITER` as different uses cases may require other delimiters.